src/averageAges.js

```diff
@@ -14,12 +14,18 @@
  *
  * @return {number}
  */
+const middleAge = (people) => {
```

Comment:
Clever decision.
However, function names should start with verb (action)
```suggestion
const getAverageAge = (people) => {
```